assets/js/plugin-check-admin.js

```diff
@@ -88,6 +103,10 @@
 			pluginCheck.actionSetUpRuntimeEnvironment
 		);
 
+		for ( let i = 0; i < data.categories.length; i++ ) {
+			pluginCheckData.append( 'categories[]', data.categories[ i ] );
+		}
+
```

Comment:
See my other comment, this should be removed. The proper list of checks is already known at this point and passed below, so we don't need to pass categories anymore.